src/components/CheckBox/CheckBox.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react"
+import { SContainer, SOuter } from "./CheckBox.styled"
+
+export type CheckboxSize = "small" | "medium" | "large"
+export type CheckboxVariant = "primary" | "secondary"
+export type CheckBoxProps = {
+  className?: string
+  label?: React.ReactNode
+  checked?: boolean
+  disabled?: boolean
+  onChange?: (checked: boolean) => void
```

Comment:
why onChange and checked props are not required?

Reply:
fixed